src/engine.cpp

```diff
@@ -18,31 +27,26 @@ bool RandomEngine::getSymbolStatus(){
     return symb;
 }
 std::string RandomEngine::getString(){
-    int mod=62;
-    if(getSymbolStatus()){
-        mod=72;
-    }
-    //Random number generation:
-    char seq[100];
-    std::ifstream file("/dev/urandom", std::ios::in | std::ios::binary);
-    if(!file) {
-        std::cout << "Error opening file\n";
-    }
-    file.read(seq, sizeof(char)*100);
-    file.close();
-    std::seed_seq seed(seq, seq+100);
-    std::mt19937_64 gen(seed);
-    std::uniform_int_distribution<int64_t> dist(0, mod);
-    //password generation:
-    std::string alpha_num="ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
-    std::string symbols="!@#$%^&*-_";
-    std::string password;
-    if(getSymbolStatus()==true){
-        alpha_num+= symbols;
+        //password generation:
+        std::string password;
+        std::seed_seq seed(seq, seq+100);
+        std::mt19937_64 gen(seed);
+        if(symb){
```

Comment:
use this check in the constructor and not here.
Understand that constructor is responsible for initializing correct variables depending on the situation.